registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/resources/bootstrap.properties

```diff
@@ -9,9 +9,13 @@ vertx.cluster.configuration=${spring.cloud.config.uri}/*/${spring.profiles.activ
 #management.security.enabled=false
 #disabling health check so that client doesnt try to load properties from sprint config server every
 # 5 minutes (should not be done in production)
+#UserNotification api version
+mosip.regproc.user.notification.api.version=1.0
```

Comment:
I don't see the property of workflow action api for the same api version (mosip.regproc.workflow-manager.action.version), do we really nead this here ?

Reply:
removed